{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/pagination/pagination.html

```diff
@@ -0,0 +1,38 @@
+{% if page_obj.has_other_pages() %}
+  <nav class="pgn-Pagination">
```

Comment:
An `aria-label` of `You are on page {{ page_obj.number }}` would be nice for screen readers.